js/main.js

```diff
@@ -0,0 +1,41 @@
+var $photoUrl = document.querySelector('#PhotoURL');
+var $userIMG = document.querySelector('#user-image');
+var $Title = document.querySelector('#Title');
+var $Notes = document.querySelector('#Notes');
+var $submit = document.querySelector('#submit');
+
+var url = $photoUrl.addEventListener('input', function (event) {
+  url = event.target.value;
+});
+var title = $Title.addEventListener('input', function (event) {
+  title = event.target.value;
+});
+var notes = $Notes.addEventListener('input', function (event) {
+  notes = event.target.value;
+});
```

Comment:
You shouldn't do this on each `input` event, you should just collect these values when the `submit` event occurs in the `submit` event handler.